extras/gizmo/gizmo.js

```diff
@@ -165,12 +165,17 @@ class Gizmo extends EventHandler {
     _scale = 1;
 
     /**
-     * Internal version of coordinate space. Defaults to {@link WORLD_COORD_SPACE}.
+     * Internal version of coordinate space. Can be:
+     *
+     * {@link GIZMO_LOCAL}
```

Comment:
the `Can be` part needs to be on the public setter, and not on a private property.

Reply:
including the default and all that. Here just a type is needed.